api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -60,11 +60,10 @@
 	public void purgeConcept(Concept concept) throws DAOException;
 	
 	/**
-	 * Get a ConceptComplex. The Concept.getDatatype() is "Complex" and the Concept.getHandler() is
-	 * the class name for the ComplexObsHandler key associated with this ConceptComplex.
+	 * Get the concept complex
```

Comment:
The previous text here was at least somewhat informative. Might suggest this text instead:

```
Get a ConceptComplex for the given concept id.

A ConceptComplex is a concept that has a datatype of `complex` and an associated ComplexObsHandler class that handles saving and loading the data associated with this concept.
```

Reply:
Thanks @ibacher